org.civicrm.reqangular/src/common/services/api/relationship.api.js

```diff
@@ -18,15 +18,16 @@ define([
       /**
        * Returns the list of relationships
        *
-       * @param {object} filters
-       * @param {object} pagination
-       * @param {string} sort
+       * @param  {Object}  filters
+       * @param  {Object}  pagination
+       * @param  {String}  sort
+       * @param  {Boolean} cache
        * @return {Promise}
        */
-      all: function (filters, pagination, sort, additionalParams) {
+      all: function (filters, pagination, sort, cache) {
```

Comment:
Where is this method currently used? Anywhere `additionalParams` passed?

Reply:
The relationship model is only being used in the PCHR-3369 epic branch. You can find the only usage here: https://github.com/compucorp/civihr-tasks-assignments/blob/7f8e6aa1630dfb3451577c015bd9de00018aff60/uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.controller.js#L363-L373

The `additionalParams` was not being used and was only added to follow the order of the `getAll` method of the API file.

Reply:
OK!